_archiv/2022/classes/essParty.html

```diff
@@ -0,0 +1,14 @@
+---
+title: "ESS Party (Social Evening Out)"
+title-short: ESS Party
+label: ess-party
+lang: deutsch/englisch
+costs: 0
```

Comment:
Warum schreibst Du hier nicht einfach "Teilnahme auf eigene Kosten." rein und sparst Dir dann die ganzen if-Abfragen?

Reply:
Der Wert in costs muss eine Zahl sein, weben der Berechnung. Könnte man schöner machen, ich weiß...